helix-tui/src/backend/crossterm.rs

```diff
@@ -54,6 +56,10 @@ impl Capabilities {
                     || t.extended_cap("Su").is_some()
                     || vte_version() >= Some(5102)
                     || matches!(term_program().as_deref(), Some("WezTerm")),
+                reset_cursor_command: t
+                    .utf8_string_cap(termini::StringCapability::CursorNormal)
+                    .unwrap_or("\x1B[0 q")
+                    .to_string(),
```

Comment:
On unic thus seems fine but does this work on windows?

Reply:
Ah good catch, I just tried it out and it doesn't work on windows since the capability will be set to an empty string instead.